index.js

```diff
@@ -995,7 +995,8 @@ module.exports = function(createParams) {
                             cause: err,
                             params: {
                                 method,
-                                ...errorParams
+                                ...errorParams,
+                                ...(err.params ?? {})
```

Comment:
`...err.params` will do the same thing as `...(err.params ?? {})`

Reply:
fixed